package.json

```diff
@@ -23,11 +23,14 @@
   },
   "homepage": "https://github.com/ttstk/build#readme",
   "dependencies": {
-    "@matanlurey/tts-save-files": "git+https://github.com/matanlurey/tts-save-format.git"
+    "@matanlurey/tts-save-files": "git+https://github.com/matanlurey/tts-save-format.git",
+    "glob": "^7.1.6"
   },
   "devDependencies": {
-    "@babel/preset-typescript": "^7.10.4",
```

Comment:
hmm, I'm not sure we should be removing this, shouldn't we be leveraging both? Do you have an article to cite here?